compiler/rustc_error_codes/src/error_codes/E0373.md

```diff
@@ -50,3 +50,41 @@ fn foo() -> Box<Fn(u32) -> u32> {
 
 Now that the closure has its own copy of the data, there's no need to worry
 about safety.
+
+This error may also be encountered while using `async` blocks:
+
+```compile_fail,E0373,edition2018
```

Comment:
Great, this test passes now! You can still simplify it though:

```rust
use std::future::Future;

async fn f() {
    let v = vec![1, 2, 3i32];
    spawn(async { //~ ERROR E0373
        println!("{:?}", v)
    });
}

fn spawn<F: Future + Send + 'static>(future: F) {
    unimplemented!()
}
```

Otherwise this change LGTM.

Reply:
I've changed the code snippet to yours in my latest commit